Examples/SnowplowSwiftCarthageDemo/Cartfile.resolved

```diff
@@ -0,0 +1,3 @@
+git "file:///Users/alexbenini/dev/snowplow/snowplow-objc-tracker" "9781def3a1ace6fd27f5d9ba0871f84fa839f868"
```

Comment:
Should we perhaps exclude the Cartfile.resolved from git in the Demo appplications and let users generate it when bootstrapping the Cartfile?
Either that or we ensure the resolved default is Github rather than a local folder.